public/index-restricted.html

```diff
@@ -0,0 +1,54 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8" />
+	<link rel="icon"
+		  type="image/svg+xml"
+		  href="/vite.svg" />
+	<meta name="viewport"
+		  content="width=device-width, initial-scale=1.0" />
+	<title>Vite + Svelte + TS</title>
+</head>
+
+<body>
+	<div>
+		<span>Your plugin is available at</span>
+		<pre class="js-info-url"></pre>
+
+		<button id="button-run">run</button>
+	</div>
+
+	<hr />
+
+	<div id="app">
+		<oscd-plugin-store></oscd-plugin-store>
+	</div>
+	<script type="module">
+		import NewOSCDPlugin from "./index-restricted.js";
```

Comment:
This can't be dynamically changed, hence why need a separate html file for restricted builds.